src/SearchTableRow.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const SearchTableRow = ({ booking, selectId }) => {
+  const [isSelected, setIsSelected] = useState(false);
+  const handleProfile = id => {
+    selectId(id);
+  };
+  const handleClick = () => {
+    setIsSelected(!isSelected);
+  };
+  return (
+    <tr className={isSelected ? "bg-info" : null} onClick={handleClick}>
+      <td id="id">{booking.id}</td>
+      <td id="title">{booking.title}</td>
+      <td id="fName">{booking.firstName}</td>
+      <td id="sName">{booking.surname}</td>
+      <td id="email">{booking.email}</td>
+      <td id="roomNo">{booking.roomId}</td>
+      <td id="chekIn">{booking.checkInDate}</td>
+      <td id="checkOut">{booking.checkOutDate}</td>
```

Comment:
Nice that you add `id` here- this makes it easier to find these columns during testing later on